### ISSUE TITLE
Missing form label

### ISSUE THREAD

**Issue by egdegauperaa** [janosh/svelte-multiselect#58]
When I run WAVE chrome plugin on the solution I'm working on, I get a couple of errors that says "A form control does not have a corresponding label". The same occurs if I run it on the svelte-multiselect demo page where the code looks like this:

![image](https://user-images.githubusercontent.com/36701818/157434565-32618560-2f45-4a21-8861-7d04df55ff98.png)

I´m not entirely sure what the purpose is of that input? Maybe it can be hidden from screen readers somehow?

**Comment by janosh** [comment]
@egdegauperaa Thanks for reporting this. There's a comment in the component file about that `input`:

https://github.com/janosh/svelte-multiselect/blob/af009b56048bc94d2514ab316e669c2896f61b65/src/lib/MultiSelect.svelte#L238-L245

I'm guessing it needs have an attribute [`aria-hidden`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden) to silence your warning.

**Comment by egdegauperaa** [comment]
@janosh I´m still getting the "missing form label" error when I run Wave in Chrome.

![image](https://user-images.githubusercontent.com/36701818/159229735-e3f02fd2-0bab-41d0-aa9c-a773eb84eb6a.png)

When I run VoiceOver on my Mac the screen reader ignores that input field, so aria-hidden seems to be the correct approach to hide something from a screen reader. I guess one approach to silence the error in Wave would be to add a hidden label for that input?

**Comment by janosh** [comment]
@egdegauperaa Thanks for reporting back. I just installed WAVE and ran it on https://svelte-multiselect.netlify.app after wrapping the offending input in a `<label>` with some random text. The a11y error went away but that leaves a bunch of CSS issues with the input no longer properly hidden. Will try to tackle those tonight.

**Comment by egdegauperaa** [comment]
HI @janosh I´d try to have the label separate (not wrap the input with the label).

**Comment by janosh** [comment]
What if we add an `aria-label` instead of a `label` element? That seems to fix the error as well and doesn't require changing the markup. Took the suggestion from w3 docs: [Approach 2: Using aria-label](https://www.w3.org/WAI/tutorials/forms/labels)

But I wonder what the value for this attribute should be. Is it ok just to set it to `"should be ignored"`.

**Comment by egdegauperaa** [comment]
That sounds good, and maybe "Label to be ignored"?
